docs/source/setup_lora.rst

```diff
@@ -72,7 +75,10 @@ Modifying the LoRa parameters
 
 -  After the registration was completed, the LoRa parameters can be transferred to the program.
 -  They can be set up at the configuration site of the Geiger counter (see above).
--  Go through the configuration site until the settings of the LoRa  parameters are displayed. Type in the 3 parameters from the TTN console (**APPEUI, DEVEUI, APPKEY**). They can be find in your TTN account for each device (see above). The HEX values must be entered **without** spaces as they appear in the TTN or other devices.
+-  Go through the configuration site until the settings of the LoRa  parameters are displayed. 
+   Type in the 3 parameters from the TTN console (**APPEUI, DEVEUI, APPKEY**). 
+   They can be find in your TTN account for each device (see above). 
```

Comment:
found

Reply:
Habe da nur die Zeilenumbrüche geändert, da ich vermute, dass es sich dann leichter übersetzen lässt.

Reply:
vor dem uebersetzen sollten die originalstrings stimmen, daher: find -> found

Reply:
Ich habe jetzt eh nochmal committed und einige Rechtschreibfehler aus unrelated files gefixed.

Reply:
Außerdem habe ich einen Aufräum-Issue #383 angelegt, denn gleichen Text in mehreren Dateien zu pflegen fand ich an der Stelle anstrengend.